boards/arm/particle_xenon/particle_xenon.dts

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2018 Endre Karlson
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "particle xenon";
+	compatible = "particle,particle_xenon", "nordic,nrf52840-qiaa",
+			 "nordic,nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		user_led: led_0 {
+			gpios = <&gpio1 12 0>;
+			label = "User LED";
+		};
+		status_green: led_1 {
+				gpios = <&gpio1 14 0>;
```

Comment:
This does not match the documentation on https://docs.particle.io/datasheets/mesh/xenon-datasheet/ (it should be `&gpio0`). Similarly, pins for `spi1` and for `spi0` (CS) are not matching the mentioned documentation. Is this a different board?

Reply:
Agree, these should be on `gpio0`.

Also, could you re-order these to the order "led1=red, led2=green, led3=blue" matching the pin assignment order and the common order of RGB?  Or does "GBR" match some Particle API?

Reply:
@anangl not sure about `spi` CS reference.  P0.31 is `SPI_SS` = "slave select" = CS.  I believe the other SPI pins are also correct.  What are we missing?